static/src/javascripts/projects/common/modules/commercial/epic/iframe-epic-utils.js

```diff
@@ -116,18 +123,40 @@ const sendFontsToIframe = (
     iframe.contentWindow.postMessage(message, '*');
```

Comment:
Minor: since we know the `targetOrigin` we may as well set it?



https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage